### PR TITLE
add allow pets check box to be able to filter shelters quickly

### PR DESCRIPTION
**Pull request by roozbeh**

### CODE REVIEW

File: app/views/shelters/_form.html.erb [hunk 1]
```diff
@@ -63,6 +63,12 @@
     <%= f.label :source %>
     <%= f.text_field :source %>
 
+    <%= f.label :phone %>
+    <%= f.text_field :phone, :required => true %>
```
Comment (tdooner) [inline app/views/shelters/_form.html.erb]: Looks like you might have resolved a merge conflict incorrectly -- the phone field was added above on line 37.

File: db/schema.rb [hunk 2]
```diff
@@ -10,11 +10,10 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170909082743) do
+ActiveRecord::Schema.define(version: 20170909175503) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
-  enable_extension "pg_stat_statements"
```
Comment (tdooner) [inline db/schema.rb]: This change seems unrelated. Can you either diagnose why your local postgres doesn't have this extension installed, or put this line back before committing the code?

**Review by tdooner — CHANGES_REQUESTED** [review]
Awesome, thanks for the contribution. A couple things need another look before I can merge this though!

File: app/views/shelters/_form.html.erb (hunk 1 above)
Comment (roozbeh) [reply app/views/shelters/_form.html.erb]: Thank you for your comment. Updated.

File: db/schema.rb (hunk 2 above)
Comment (roozbeh) [reply db/schema.rb]: Thanks for your comment. The extension added to psql, and by a `rake db:migrate` it added back by rake.

**Review by tdooner — APPROVED** [review]
Looks good.
